meteor/utils.py

```diff
@@ -12,6 +12,7 @@
 class MapLabels:
     amplitude: str
     phase: str
+    error: str | None = None
```

Comment:
wait, is error = uncertainty?

Reply:
hmmm yes `uncertainty` (or `std`?) is a better name!